src/dialects/oracle/query.js

```diff
@@ -56,6 +56,35 @@ export class OracleQuery extends AbstractQuery {
     }
     return execOpts;
   }
+
+  /**
+   * convert binding values for unsupported
```

Comment:
This looks good but can you check if it is feasible at the place I have pointed out? We're re-iterating the map and that sounds like an overhead.

Reply:
I think we have to do it in query, not DataType, because users can pass bigint bind parameters themselves without going through DataTypes when using raw queries, like with the following:

```ts
sequelize.query('SELECT * FROM users WHERE id = $id', {
  bind: {
    id: 123n
  }
});
```

Reply:
@ephys Yes, that's right. Having it here will not break it when using a raw query.

Reply:
Thank you so much for taking the time to review this by the way :)

Reply:
Sure, Happy to help. :)